department-of-veterans-affairs/caseflow
# Intake | Handle missing VACOLS issue

connects #14592

### Description
There was an issue that got opted-in to AMA, and then subsequently was deleted in VACOLS.  This is causing an error with the edit page loading.

This PR adds handling for when an opted-in issue goes missing in VACOLS.

### Testing Plan
1. Create a request issue that's connected to a legacy issue, but do not create the legacy issue in test data
1. Confirm that the edit page and confirmation pages still load.

### User Facing Changes
- Before "white screen"
- After page loads, reference to opted-in VACOLS issue is removed
